src/pages/index.tsx

```diff
@@ -3,55 +3,187 @@ import Gnb from '@/components/layout/gnb/Gnb'
+            <div className={`${styles.homepagelogo} inline-block text-center`}>
+              <Image
+                src="/assets/image/desktop.svg"
+                alt="Taskify Logo"
+                fill
+                priority
+                className={styles.image}
+              />
             </div>
-          </div>
-        </section>
-
-        <section className="flex justify-center w-full">
-          <span className={`${styles.linkcardlabel} mr-auto`}>
-            생산성을 높이는 다양한 설정⚡
-          </span>
-        </section>
 
-        <section className="flex flex-row text">
-          <div></div>
-        </section>
+            <h1
+              className={`font-bold text-white ${styles.homepagemaintext} ${styles.line}`}
+            >
+              새로운 일정 관리
+              <span className={` ${styles.homepagespan} ${styles.line}`}>
+                Taskify
+              </span>
+            </h1>
+            <Link href={'/login'}>
+              <Button
+                variant="login"
+                size="large"
+                isActive={true}
+                className={`${styles.loginbutton}`}
+              >
+                로그인 하기
+              </Button>
+            </Link>
+          </section>
+        </AnimatedSection>
+        <AnimatedSection delay={0.2}>
+          <section className={`${styles.pointer1}  flex-col`}>
+            <div className={`${styles.pointer1image}`}>
+              <Image
+                src="/assets/image/landing1.svg"
+                alt="landingpage image"
+                fill
+                priority
+              />
+            </div>
+            <span className={`${styles.pointer1text}`}>Point1</span>
+            <div className={`${styles.pointer1maintext}`}>
+              <div>일의 우선 순위를</div>
+              <div>관리하세요</div>
+            </div>
+          </section>
+        </AnimatedSection>
+        <AnimatedSection delay={0.2}>
+          <section className={`${styles.pointer2} flex flex-col `}>
+            <div className={`${styles.pointer2image}`}>
+              <Image
+                src="/assets/image/landing2.svg"
+                alt="landingpage image"
+                fill
+                priority
+              />
+            </div>
+            <div className={`${styles.pointer2box}`}>
+              <span className={`${styles.pointer2text}`}>Point2</span>
+              <div className={`${styles.pointer2maintext}`}>
+                <div>해야 할 일을</div>
+                <div>등록하세요</div>
+              </div>
+            </div>
+          </section>
+        </AnimatedSection>
+        <AnimatedSection delay={0.2}>
+          <div className="flex justify-center w-full">
+            <span className={`${styles.linkcardlabel} mr-auto`}>
+              생산성을 높이는 다양한 설정⚡
+            </span>
+          </div>
+        </AnimatedSection>
+        <AnimatedSection delay={0.2}>
+          <section className={`${styles.cardListWrapper}`}>
```

Comment:
참조하는 값 자체가 문자열이라 다시 문자열로 래핑할 필요가 없을것 같습니다!
```suggestion
          <section className={styles.cardListWrapper}>
```

Reply:
이것도 참고하겠습니다!